src/Plugin/Core/InstallerScripts/AutoloadConnector.php

```diff
@@ -53,12 +57,19 @@ public function run(Event $event): bool
 
         $composerConfig = $composer->getConfig();
         $localRepository = $composer->getRepositoryManager()->getLocalRepository();
-        $package = $localRepository->findPackage('typo3/cms', new EmptyConstraint());
+        $package = $localRepository->findPackage('typo3/cms', class_exists(EmptyConstraint::class) ? new EmptyConstraint() : new MatchAllConstraint());
```

Comment:
can be simplified to $localRepository->findPackage('typo3/cms', '*');

Reply:
n1, adapted it